assignment.md

```diff
@@ -11,63 +11,76 @@ If the version control concepts or `git` commands are still confusing to you, co
+      >"added ch2 assignment" 
 
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > Git stash saves changes I made locally elsewhere that's not on my repo, and it restores my repository to the last commit, any uncommitted changes I had were reversed when the file was opened again
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > 3a13bb9 (refs/stash) WIP on master: 540a90d added week5 
      - c. Look up what `git stash drop` and `git stash pop` does, either Google or use `--help`. Run `git stash drop` to undo what you did with `stash`, in what scenario might `stash`-ing, `pop`-ing, and `drop`-ing be useful?
-       > Answer here
+       > Pop less you make the stashed changes to multiple brances and drop lets you delete any of the stashes you no longer need. Stash is helpful when you're working on multiple files and branches and need to switch content but don't neccesarily want to commit the changes. you can shelf them and apply them later.
 
  6. One common mistake when learning `git` is to commit large files that should not be managed by `git` or adding sensitive information like security keypairs for Amazon Cloud Services. Navigate back to your assignment repository and try adding a random text file to the repository (use `touch`), making some modifications and commits to your repository, and deleting that file with `rm`. But deleting with `rm` will not delete that file's recorded git history, therefore we need to do a little more. You may want to look at [this](https://help.github.com/articles/removing-sensitive-data-from-a-repository/))
 
-    > Perform the commits and deletion, this question will be graded based on your commit history on GitHub. I should see commits for when you added your file and made modifications, and I should see a commit for when you deleted the files. 
+    > I did this with you on the file "plant"
 
 
  7. Like many command line tools, `git` provides a configuration file (or dotfile) called `~/.gitconfig`. Create an alias in `~/.gitconfig` so that when you run `git graph`, you get the output of `git log --all --graph --decorate --oneline`. You can do this by directly modifying the `~/.gitconfig` file or by using the `git` command line functionality.
     ```bash
-    # Your alias entry or git config command here
+    # [alias]                                                                                                                                  doggo = log --all --graph --decorate --oneline 
```

Comment:
I would've preferred it if it was `git graph` not `git doggo`, but whatever floats your boat...